install/HowtoSetupContiv.md

```diff
@@ -30,41 +30,51 @@ Usage: netmaster [OPTION]...
         prints current version
 ```
 
-## How to start netplugin ? 
+## How to start netplugin ?
 
 You start netplugin on each node in your cluster.
 
 ```
 netplugin --help
-Usage of netplugin:
-  -cluster-store string
-        state store url (default "etcd://127.0.0.1:2379")
-  -config string
-        plugin configuration. Use '-' to read configuration from stdin
-  -ctrl-ip string
-        Local ip address to be used for control communication
-  -debug
-        Show debugging information generated by netplugin
-  -host-label string
-        label used to identify endpoints homed for this host, default is host name. If -config flag is used then host-label must be specified in the the configuration passed. (default "aci-testbed-swarm-1")
-  -json-log
-        Format logs as JSON
-  -plugin-mode string
-        plugin mode docker|kubernetes (default "docker")
-  -syslog string
-        Log to syslog at proto://ip:port -- use 'kernel' to log via kernel syslog
-  -version
-        Show version
-  -vlan-if string
-        VLAN uplink interface
-  -vtep-ip string
-        My VTEP ip address
+NAME:
+   netplugin - Contiv netplugin service
+
+USAGE:
+   netplugin [global options] command [command options] [arguments...]
+
+VERSION:
+
+Version: 9dffbee-unsupported
+GitCommit: 9dffbee-unsupported
+BuildTime: 11-10-2017.16-09-00.UTC
```

Comment:
maybe use descriptions instead of actuals?
might want indication near the `netplugin --help` that some values from help are calculated and will be represented below with \<whatever style>

Reply:
done